docs/getting-started.md

```diff
@@ -10,10 +10,12 @@ contains a Misk web app with the requisite dependencies.
 
 ## Start the service
 
+If you have not done so already, install and activate hermit: https://cashapp.github.io/hermit/
```

Comment:
Probably want a little more description of activating the Hermit environment. Also wonder if directing people to use `bin/gradle` directly is good.

Reply:
You mean use show the commands as bin/gradle instead of just gradle?